schemas/oic.if.linkupdate-schema.json

```diff
@@ -0,0 +1,72 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description" : "Copyright (c) 2018 Open Connectivity Foundation, Inc. All rights reserved.",
+    "id": "http://openconnectivityfoundation.github.io/core/schemas/oic.if.linkupdate-schema.json#",
+    "title": "Linkupdate Format",
+    "definitions": {
+      "linkupdate-schema": {
+        "type": ["object", "array"],
+        "properties": {
```

Comment:
Should "rt" and "if" be included here?  They are included in the examples in the CR for Bug 1229.